src/sentry/replays/consumers/recording/process_recording.py

```diff
@@ -216,6 +218,13 @@ def terminate(self) -> None:
         self.close()
         self.__threadpool.shutdown(wait=False)
 
+    def _configure_sentry_scope(self, message_dict, offset):
+        with configure_scope() as scope:
+            scope.set_tag("replay_id", message_dict["replay_id"])
+            scope.set_tag("project_id", message_dict["project_id"])
+            scope.set_tag("offset", offset)
```

Comment:
Is offset the segment that is duplicated?

Reply:
no, something with kafka -- actually don't think we need it here, removed. and the duplicated segment will be in the breadcrumbs.